src/App.tsx

```diff
@@ -1,204 +1,79 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
-
+//#region imports
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { ErrorNotifications } from './types/ErrorNotifications';
+import { FilterBy } from './types/FIlterBy';
+//#endregion
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  //#region states
+  const [todos, setTodos] = useState<Todo[] | null>(null);
```

Comment:
```suggestion
  const [todos, setTodos] = useState<Todo[]>([]);
```
default should be array